go/ql/test/library-tests/semmle/go/frameworks/Spew/TaintFlows.ql

```diff
@@ -1,28 +1,18 @@
 import go
+import TestUtilities.InlineFlowTest
 
-class UntrustedFunction extends Function {
-  UntrustedFunction() { this.getName() = ["getUntrustedString", "getUntrustedStruct"] }
-}
-
-class UntrustedSource extends DataFlow::Node, UntrustedFlowSource::Range {
-  UntrustedSource() { this = any(UntrustedFunction f).getACall() }
-}
-
-class SinkFunction extends Function {
-  SinkFunction() { this.getName() = "sinkString" }
-}
-
-class TestConfig extends TaintTracking::Configuration {
-  TestConfig() { this = "testconfig" }
-
-  override predicate isSource(DataFlow::Node source) { source instanceof UntrustedFlowSource }
+module TestConfig implements DataFlow::ConfigSig {
+  predicate isSource(DataFlow::Node source) {
+    source =
+      any(Function f | f.getName() = ["getUntrustedString", "getUntrustedStruct"])
+          .getACall()
+          .getResult()
```

Comment:
I note that the test previously ranged over all of `UntrustedFlowSource` but now is an inlined `UntrustedSource`. Do we miss out anything important by not having this range over all of `UntrustedFlowSource` anymore?

Reply:
I don't think so - we're just trying to match the two functions used as sources in the test files. I can only imagine was written like this to be more like a real source definition.